packages/react/src/components/_CheckboxRadioTemplate/CheckboxRadioTemplate.module.css

```diff
@@ -77,18 +77,25 @@
   color: var(--label-color);
 }
 
+.labelAndHelpText {
+  display: inline-flex;
+  flex-direction: row;
+  gap: var(--gap);
+  align-items: center;
+}
+
 .description {
   color: var(--description-color);
 }
 
 @supports not selector(:has(:focus-visible)) {
-  .template:not(.disabled):focus-within {
+  .template:not(.disabled):not(:has(button:focus-visible)):focus-within {
```

Comment:
Dette er koden som kjøres hvis nettleseren ikke støtter `:has` (Firefox), så dette må nok løses på en annen måte. Hvis det blir vanskelig å finne et alternativ for Firefox, bør vi kanskje hjelpe til med Javascript (se `Select`, der har vi en funksjon som som heter `usingKeyboard` eller noe slikt).

Reply:
Jeg har ikke klart å finne en løsning på å få visningen av fokus oppføre seg likt uten å bruke :has(), så jeg lener mot å bare ikke håndtere det for nettlesere som ikke støtter :has(). Forskjellen er slik:
Med :has()
![Screen Recording 2023-02-01 at 11 50 24](https://user-images.githubusercontent.com/32294735/216023303-00fbc3e0-4bab-4563-95b3-7c41ddb9420c.gif)
Uten :has()
![Screen Recording 2023-02-01 at 11 51 49](https://user-images.githubusercontent.com/32294735/216023437-505596e4-ed20-4a47-afcd-926abde74d94.gif)

Reply:
@mrosvik, hva tenker du? Brukere av Firefox vil få den nederste visningen nå, men det er grunn til å tro at også Firefox legger til støtte for dette i nær fremtid. Alternativet er å hacke til dette med Javascript, og det er en større jobb.

Reply:
Jeg tror ikke fokus-visningen i firefox er et stort problem, men mulig det vil bli registrert som en wcag-feil, ettersom det kan være noe uklart om det er label eller helptext som er i fokus. Vet du @Febakke ?

Det jeg er mer bekymret for her er at helptext-symbolet er veldig likt utformet som selve radioknappen. (Det er to sirkler på hver side av teksten.) Helptext funker fint sammen med Legend, hadde det vært nok å hatt mulighet for å legge det på der? Har vi noen konkrete caser der det er behov for hjelpetekst ved siden av et svaralternativ? Det er jo også mulighet for description under hver radioknapp, og OED har også hatt behov for å vise ganske mye tekst under description. I den anledning tenkte vi på om det kunne være et valg å aktivere at description kun vises på radioknappen du har valgt. Slik som dette: https://design.entur.org/komponenter/skjemaelementer/radio-panel (Når du trykker på den siste her så vises det en description) Ville dette evt. kunne dekt noen behov?

Reply:
Jeg kan ikke se fra [2.4.7 Focus visible](https://www.w3.org/WAI/WCAG22/Understanding/focus-visible.html) at det er et direkte brudd, selv om det er uheldig. Enig i at vi kan registrere en bug på det og se på det senere. 

Jeg vet at SSB har bedt om å kunne plassere en hjelpetekst slik. Det er jo helt avhengig av hvilken type informasjon det er om det passer som en description eller hjelpetekst. Men! Hvis man velger å plassere de horisontalt, da blir dette veldig rart. Det er i det minste noe vi burde skrive om i retningslinjene!